### ISSUE TITLE
Support material design theme

### ISSUE THREAD

**Issue by kkimdev** [PistonDevelopers/conrod#491]
This is more like a wish-list item :)

https://www.polymer-project.org/1.0/components/paper-elements/demo.html

**Comment by cyplo** [comment]
Hi !
The link does not seem to work for me, could you take a look ?
Thanks !

**Comment by kkimdev** [comment]
Yeah the link seems dead. Here is another one https://ebidel.github.io/material-playground/ . And here is the spec https://www.google.com/design/spec/material-design/introduction.html

**Comment by mitchmindtree** [comment]
I love the flat and clear visual aesthetic of the material design! Although having used google play music for a couple years, I find the animations can be painfully slow/unnecessary. Either way, it should be very possible to begin implementing a material design widget set using conrod in its current state. Whether we add them to the conrod crate itself or update existing widgets is another question. I think they'd be best in a `conrod_material` crate or something along those lines for now at least.
